app/src/utils/ipfs.ts

```diff
@@ -1,18 +1,9 @@
-import { create } from 'ipfs-http-client';
+import { createIpfs } from '@dgrants/utils/src/ipfs';
 import { GrantMetadata } from '@dgrants/types';
 
-const config = {
-  storageEndpoint: 'https://ipfs-api.dev.fleek.cool',
-  storageHeaders: {
-    Authorization: `v2 ${import.meta.env.VITE_FLEEK_STORAGE_API_KEY}`, // or Bearer <JWT> or public <AppKey>
-  },
-  retrievalEndpoint: 'https://ipfs-dev.fleek.co/ipfs',
-};
+const RetrievalEndpoint = 'https://ipfs-dev.fleek.co/ipfs';
```

Comment:
Nit on variable naming
```suggestion
const retrievalEndpoint = 'https://ipfs-dev.fleek.co/ipfs';
```

Reply:
Fixed! I do this out of habit from Swift conventions. Just for reference, here's what the AirBnB guide [says about constant naming](https://github.com/airbnb/javascript/#naming--uppercase) in JavaScript: use ALL_CAPS_SNAKE_CASE, but only if it's exported.